packages/webapp/src/utils/credits-purchase-constants.js

```diff
@@ -1,12 +1,30 @@
-const routes = {
-  ESTIMATOR_CREDITS_INDIVIDUAL_ORG: '/credits-purchase/applying-individual-organisation'
-}
+// Routes constants
+const CREDITS_APPLICATION_BY_INDIVIDUAL_OR_ORGANISATION = 'credits-purchase/applying-individual-organisation'
+const CREDITS_PURCHASE_ORDER = 'credits-purchase/credits-purchase-order'
+// ./Routes constants
+
+// RedisKeys constants
+const CREDITS_PURCHASE_USER_TYPE = 'credits-purchase-user-type'
+const WILL_CREDITS_PO_IN_USE = 'will-credits-po-in-use'
```

Comment:
Add prefix `CREDITS_PURCHASE` to all routes, views and redis keys for the credits purchase journey